### DISCUSSION TITLE
Change font size of GUI?

### DISCUSSION THREAD

**Discussion by GabZach** [dfranx/SHADERed#264] [Q&A]
Is it possible to change the font size of the GUI?
The reason why i want to do that:
when I present SHADERed with a projector connected to my Mac, SHADERed uses a huge font size, so that one cannot see complete lines of GLSL code.
(The font size of all other programs is fine , in fact it does not change with or without a projector connected to the Mac.)

**Comment by dfranx** [comment]
Yeah, it's possible to change the font size... First go to options: `Window` &rarr; `Options`.

You should first try turning off the "DPI aware" option (`General` &rarr; scroll down below &rarr; find the `DPI aware` option) as that may cause the font to become huge when switching from Mac to projector..

But if you actually want to change font size, above the `DPI aware` option is the UI font size option. `Editor` &rarr; `Font size` to change the size of the font in the text editor.
